### PR TITLE
*: fix clippy warning

### PR DESCRIPTION
**Pull request by overvenus**
This PR fixed some clippy warnings.

### CODE REVIEW

File: proto/metrics.rs [hunk 1]
```diff
@@ -2711,7 +2711,7 @@ impl ::protobuf::ProtobufEnum for MetricType {
     }
 
     fn values() -> &'static [Self] {
-        static values: &'static [MetricType] = &[
```
Comment (siddontang) [inline proto/metrics.rs]: can the rust version used in TiKV support removing static?

Comment (overvenus) [reply proto/metrics.rs]: Yes.
```
neil:rust-prometheus/ (ov/clippy) $ rustc -V
rustc 1.21.0-nightly (215e0b10e 2017-08-08)
neil:rust-prometheus/ (ov/clippy) $ cargo check
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
```

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay

File: src/desc.rs [hunk 2]
```diff
@@ -18,23 +18,27 @@ use fnv::FnvHasher;
 use metrics::SEPARATOR_BYTE;
 
 use proto::LabelPair;
-use std::ascii::AsciiExt;
 use std::collections::{BTreeSet, HashMap};
 use std::hash::Hasher;
 
+// TODO: use `char::is_ascii` instead once it landed in the stable rust.
+fn is_ascii(c: char) -> bool {
+    c as u8 & 128 == 0
```
Comment (BusyJay) [inline src/desc.rs]: This is wrong. Should be `c as u32 < 128`.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**